Description: pin the AppArmor feature set to the one shipped by the apparmor package
 .
 Let's smooth UX on kernel upgrades and allow ourselves to update the AppArmor
 policy in a relaxed manner.
Bug-Debian: https://bugs.debian.org/879585
Forwarded: not-needed
Author: intrigeri <intrigeri@debian.org>

--- a/parser/parser.conf
+++ b/parser/parser.conf
@@ -59,3 +59,7 @@
 ## Adjust compression
 #Optimize=compress-small
 #Optimize=compress-fast
+
+## Pin feature set (avoid regressions when policy is lagging behind
+## the kernel)
+features-file=/usr/share/apparmor-features/features
